# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../microcode_ctl/no_install_rc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2003-04-29 09:46:06.000000000 +0200
+++ ./Makefile	2003-04-29 09:46:22.000000000 +0200
@@ -50,18 +50,6 @@
 	$(INS) -m 644 $(MANPAGE) $(DESTDIR)$(MANDIR)
 	gzip -9f $(DESTDIR)$(MANDIR)/$(MANPAGE)
 
-	$(INS) -m 755 $(RCFILE) $(DESTDIR)$(RCFILETO)/$(RCFILEFINAL)
-
-ifndef DESTDIR
-		chkconfig --add $(RCFILEFINAL)
-else
-		echo "MAKE: Skipping chkconfig operation (rpm build?)"
-endif
-
-device:
-	mkdir -p $(DESTDIR)/dev/cpu
-	mknod $(DESTDIR)/dev/cpu/microcode c 10 184
-
 uninstall:
 ifndef DESTDIR
 	chkconfig --del $(RCFILEFINAL)
